### ISSUE TITLE
'jackInTheBox' is not included in the latest 3.5.2 NPM package.

### ISSUE THREAD

**Issue by ben-yip** [daneden/animate.css#737]
While the version message is correctly 3.5.2

**Comment by awais786327** [comment]
just put this code into your css file and use it by simply calling class name 
```
<div class="animated jackInTheBox"></div>
```
```
.jackInTheBox {
    animation-name: jackInTheBox
}

@keyframes jackInTheBox {
    0% {
        opacity: 0;
        transform: scale(.1) rotate(30deg);
        transform-origin: center bottom
    }
    50% {
        transform: rotate(-10deg)
    }
    70% {
        transform: rotate(3deg)
    }
    to {
        opacity: 1;
        transform: scale(1)
    }
}

@keyframes rollIn {
    0% {
        opacity: 0;
        transform: translate3d(-100%, 0, 0) rotate(-120deg)
    }
    to {
        opacity: 1;
        transform: none
    }
}
```

**Comment by eltonmesquita** [comment]
It'll be included in the next version, 3.6.0 as it's a new animation. I'll ask Daniel to publish it as soon as it gets released.

**Comment by eltonmesquita** [comment]
Keeping this conversation on #728.
